lambdas/data-migration2/src/storeErrors.ts

```diff
@@ -0,0 +1,34 @@
+import moment from 'moment';
+
+const { s3 } = require('@cumulus/aws-client/services');
+const fs = require('fs');
+/**
+ * Store migration errors JSON file on S3.
+ *
+ * @param {Object} params
+ * @param {string} params.bucket - Name of S3 bucket where file will be uploaded
+ * @param {string[]} params.filename - Write Stream filename
+ * @param {string} params.recordClassification - Classification of record
+ * @param {string} params.stackName - User stack name/prefix
+ * @param {string | undefined} params.timestamp - Timestamp for unit testing
+ * @returns {void}
+ */
+export const storeErrors = async (params: {
+  bucket: string,
+  filename: string,
+  recordClassification: string,
```

Comment:
Nit: I'm gonna recommend `migrationName` as this variable name vs `recordClassification`, since I think that's a clearer explanation of what it is